client/app/dbaas/metrics/dashboard/metrics-dashboard.controller.js

```diff
@@ -67,11 +80,19 @@
             }
         }
 
+        initActions () {
+            this.actions.autorenew = this.ControllerHelper.navigation.getUrl("renew", { serviceName: this.serviceName, serviceType: "METRICS" });
+            this.actions.contacts = this.ControllerHelper.navigation.getUrl("contacts", { serviceName: this.serviceName });
+        }
+
         computeUsage (value, total) {
             return value / total * 100;
         }
 
         displayUsage (value, total) {
+            if (!value && !total) {
```

Comment:
can value really be undefined / null?  total?

Reply:
Yap if you pass a wrong service name in url. This is to avoid NaN returned.